ui/src/clockface/components/inputs/multipleInput/MultipleRows.tsx

```diff
@@ -20,10 +20,10 @@ interface RowsProps {
 const Rows: SFC<RowsProps> = ({tags, onDeleteTag, onChange}) => {
   return (
     <div className="input-tag-list">
-      {tags.map(item => {
+      {tags.map((item, index) => {
         return (
           <Row
-            index={tags.indexOf(item)}
+            index={index}
```

Comment:
what dis about?

Reply:
Truthfully this was me working on:

https://github.com/influxdata/honeybadger-errors/issues/19

Since I believe that the issue is resolved and won't be pushing up the change, I figured it was small and straightforward enough update that simplified the component's logic. I can remove it from the PR if needed

Reply:
hrmm.  it does seem like a reasonable change, but I also think we shouldn't fix it if we don't know its not broken? I'm not sure.

Reply:
I hear ya, I'll go ahead and revert it